tests/cpp-tests/Classes/ExtensionsTest/CocoStudioActionTimelineTest/ActionTimelineTestScene.cpp

```diff
@@ -66,11 +66,11 @@ void TestActionTimeline::onEnter()
 {
     ActionTimelineBaseTest::onEnter();
 
-    Node* node = CSLoader::createNode("ActionTimeline/DemoPlayer.csb");
-    ActionTimeline* action = CSLoader::createTimeline("ActionTimeline/DemoPlayer.csb");
+    Data data = FileUtils::getInstance()->getDataFromFile("ActionTimeline/DemoPlayer.csb");
+    Node* node = CSLoader::createNodeWithDataBuffer(data);
+    ActionTimeline* action = CSLoader::createTimelineWithDataBuffer(data, "ActionTimeline/DemoPlayer.csb");
```

Comment:
If add filename to data, we need define a new data structure and construct it before calling the function, it will increase the cost for user start using it, not that friendly. Key can be any strings, but use full path of file name is a simple, clearly and cost less way to distinguish every item. And it has been used with old version for a long time, change it may cause error for old projects.